java/arcs/sdk/android/storage/service/StorageService.kt

```diff
@@ -61,7 +60,7 @@ import kotlinx.coroutines.runBlocking
 open class StorageService : ResurrectorService() {
     private val coroutineContext = Dispatchers.IO + CoroutineName("StorageService")
     private val scope = CoroutineScope(coroutineContext)
-    private val writeBackScope = CoroutineScope(
+    open val writeBackScope = CoroutineScope(
```

Comment:
Rather than opening up a bunch of individual variables, we should consider a single 'configuration' variable/field that's open.